admin_server/routes/auth.js

```diff
@@ -0,0 +1,146 @@
+const router = require('express').Router();
+const axios = require('axios');
+const jwt = require('jsonwebtoken');
+
+const MSG91_URL = "https://control.msg91.com/api/";
+
+//----------Send OTP--------------
+router.post('/sendotp', async (req, res) => {
+  
+  try {
+
+    const params = {
+      country: req.body.country_code,
+      mobile: req.body.mobile_no,
+      authkey: process.env.MSG91_API_KEY,
+      sender: "HLOCHN",
+      message: "Your Hello Chennai app OTP is ##OTP##"
+    };
+    
+    const { data } = await axios.post(MSG91_URL+"sendotp.php",null, { params });
+
+    if(data.type === 'error'){
+
+      res.status(400).send({
+        message: ""+data.message,
+        status: "failed"
+      });
+
+    } else {
+      
+      res.status(200).send({
+        message: "sms sent",
+        status: "success"
+      });
+
+    }
+
+  } catch (error) {
+
+    console.log(error);
+    res.status(500).send({
+      message: "internal error",
+      status: "failed",
+      error: ""+error
```

Comment:
Do we need to send the error in the response. We can log it. The error payload may be huge

Reply:
only the error message should be fine. it can be useful for debugging

Reply:
What i meant was ... status code 500 corresponds to internal server error. Do we need to specify that explicitly ??.
Also we are appending the error object in line 44.

Reply:
for the sake of consistency and ease of development, i would like to have a consistent response message for every call. 
In line 44 since i have appended with empty string (""+) only error message will be printed, not the entire object